### PR TITLE
New CSS data-type: generic family

### PR DESCRIPTION
**Pull request by estelle**
Part of https://github.com/openwebdocs/project/issues/176

enumerated values are defined in https://drafts.csswg.org/css-fonts/#generic-family-value and https://drafts.csswg.org/css-fonts/#generic-font-families

### CODE REVIEW

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle looking good! I have made a few comments, but nothing major.

File: files/en-us/web/css/generic-family/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  - : The default user interface serif font. See the definition of `serif` above.
+
+- `ui-sans-serif`
+
+  - : The default user interface sans-serif font. See the definition of `sans-serif` above.
+
+- `ui-monospace`
+
+  - : The default user interface monospace font. See the definition of` monospace` above.
```
Comment (estelle) [inline files/en-us/web/css/generic-family/index.md:57]:
```suggestion
  - : The default user interface monospace font. See the definition of `monospace` above.
```

File: files/en-us/web/css/generic-family/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  <li class="fantasy">fantasy</li>
+  <li class="system-ui">system-ui</li>
+</ul>
+```
+
+### CSS
+
+```css
+ul {
+  line-height: 2;
```
Comment (estelle) [inline files/en-us/web/css/generic-family/index.md:95]:
```suggestion
  font-size: 1.5rem;
  line-height: 2;
```

File: files/en-us/web/css/generic-family/index.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  font-family: fantasy;
+}
+.system-ui {
+  font-family: system-ui;
+}
+```
+
+### Result
+
+{{EmbedLiveSample("Examples", "500", "250")}}
```
Comment (estelle) [inline files/en-us/web/css/generic-family/index.md:120]:
```suggestion
{{EmbedLiveSample("Examples", "500", "300")}}
```

File: files/en-us/web/css/generic-family/index.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+The `<generic-family>` {{glossary("enumerated")}} type is specified using one of the values listed below:
+
+- `serif`
+
+  - : A serif is a small line or stroke attached to the end of a larger stroke in a letter. In serif fonts, glyphs have finishing strokes, flared or tapering ends. Examples include Lucida Bright, Lucida Fax, Palatino, Palatino Linotype, Palladio, and URW Palladio.
+
+- `sans-serif`
+
+  - : Glyphs have plain stroke endings, without ornamentation. Example sans-serif fonts include Open Sans, Fira Sans, Lucida Sans, Lucida Sans Unicode, Trebuchet MS, Liberation Sans, and Nimbus Sans L.
```
Comment (estelle) [inline files/en-us/web/css/generic-family/index.md:29]:
```suggestion
  - : A font without serifs; glyphs have plain stroke endings, without ornamentation. Example sans-serif fonts include Open Sans, Fira Sans, Lucida Sans, Lucida Sans Unicode, Trebuchet MS, Liberation Sans, and Nimbus Sans L.
```

**Review by chrisdavidmills — APPROVED** [review]
Yessssss, perfect. Approving!

**Pull request merged**